be/src/pipeline/task_queue.cpp

```diff
@@ -182,7 +182,7 @@ PipelineTask* MultiCoreTaskQueue::_steal_take(size_t core_id) {
         DCHECK(next_id < _core_size);
         auto task = _prio_task_queue_list[next_id].try_take(true);
         if (task) {
-            task->set_core_id(next_id);
```

Comment:
need change the origin core id seems right？

Reply:
> need change the origin core id seems right？

I think core_id of pipeline task means which core thread the task belong to and which multi core task queue the task belong to.  On the other hand,  previous_core_id of pipeline task means the origin core id.
So the task is stealed from next_id to the core_id, it should be the core_id.